core/src/main/java/org/bitcoinj/core/TransactionConfidence.java

```diff
@@ -337,23 +339,37 @@ public boolean wasBroadcastBy(PeerAddress address) {
     }
 
     /** Return the time the transaction was last announced to us. */
-    public Date getLastBroadcastedAt() {
+    @Nullable
+    public Instant getLastBroadcastedAtInstant() {
```

Comment:
You could name this `getLastBroadcastTime()` if you like.

Also, as this is a return value, using `Optional` here would probably be better.

Reply:
Good idea!